### DISCUSSION TITLE
Allow plugins/presets to add paths to `content`.

### DISCUSSION THREAD

**Discussion by crswll** [tailwindlabs/tailwindcss#8926] [Ideas]
I asked about this on Discord and don't remember if you asked me to open something up or what but here it is.

It would be cool if a plugin or a preset can add an item to the `content` array so instead of instructing on adding a path to a `node_modules` directory we can just let a plugin or preset to it.

**Comment by silasabbott** [comment]
I think this makes a lot of sense. Currently, when content is defined in `tailwind.config.js`, it overrides whatever the plugin tries to define. Thus, the user has to manually add a long `./node_modules/...` string to `config.content` to get Tailwind to pick up selectors in the external library. It would be awesome if we could eliminate that extra step and just append paths from within the plugin.

[Here's an example](https://flowbite.com/docs/getting-started/svelte/#method-1-using-flowbite-svelte) of that extra step from Flowbite's installation page
